lib/src/mt_dev.c

```diff
@@ -1608,6 +1611,11 @@ static int dev_if_init_pacing(struct mt_interface* inf) {
     if (inf->port_type == MT_PORT_VF) {
       ret = dev_init_ratelimit_vf(inf);
     } else {
+      if (inf->feature & MT_IF_FEATURE_TX_OFFLOAD_SEND_ON_TIMESTAMP) {
+        inf->tx_pacing_way = ST21_TX_PACING_WAY_TSN;
```

Comment:
Why it put under RL logical? More reasonable if we put it directly under the top of this function.
```
if ((ST21_TX_PACING_WAY_AUTO == inf->tx_pacing_way) && (inf->feature & MT_IF_FEATURE_TX_OFFLOAD_SEND_ON_TIMESTAMP)) {
  info("%s(%d), use TSN pacing\n", __func__, port);
  inf->tx_pacing_way = ST21_TX_PACING_WAY_TSN;
  return 0;
}
```

Reply:
> Why it put under RL logical? More reasonable if we put it directly under the top of this function.
> 
> ```
> if ((ST21_TX_PACING_WAY_AUTO == inf->tx_pacing_way) && (inf->feature & MT_IF_FEATURE_TX_OFFLOAD_SEND_ON_TIMESTAMP)) {
>   info("%s(%d), use TSN pacing\n", __func__, port);
>   inf->tx_pacing_way = ST21_TX_PACING_WAY_TSN;
>   return 0;
> }
> ```

Resolved by [commit](https://github.com/OpenVisualCloud/Media-Transport-Library/commit/e743943171131107a121da0346cc990ddf6748fb)